grails-app/assets/javascripts/streama/controllers/admin-fileManager-ctrl.js

```diff
@@ -9,6 +9,8 @@ angular.module('streama').controller('adminFileManagerCtrl', ['$scope', 'apiServ
 
 	$scope.activeListDisplay = 'table';
 
+	$scope.deletionBulk = [];
```

Comment:
actually, I would probably rename this array and the method that adds or removes it, Because I just realized, this is merely the selection. The selection itself has nothing to do with bulk-deletion. Think of it this way: you could also select 4, and then choose another action, like "move to different harddrive" or similar. So basically, this array and the method below is `$scope.selectedFiles = []` and `addOrRemoveFromSelection = function(){`